### DISCUSSION TITLE
Email Trigger

### DISCUSSION THREAD

**Discussion by Fabian1410** [Budibase/budibase#1761] [Ideas]
It would be nice to have a email trigger in the automation. So that the integrated smtp Server could receive email, and add them to the automation workflow (title, sender, attachments …)

**Comment by shogunpurple** [comment]
@Fabian1410 Nice idea. This is a pretty common workflow in a lot of automation platforms as well. We will be looking to revamp quite a bit of the automations platform in the coming months, so will certainly take this enhancement into account. Thanks!

**Reply by Nasnl** [reply]
Just curious to see how far along you are with this one... because indeed it would be a great feature.

**Comment by garymcnamara** [comment]
Also curious; if we had the ability to receive an e-mail, then it would make the support template much much better.

**Comment by benaunt** [comment]
The ability to receive an email, trigger a workflow, and move its data + attachments to a data source would be a huge benefit to us.
